sdk/webpubsub/Azure.Messaging.WebPubSub/README.md

```diff
@@ -47,23 +43,23 @@ var serviceClient = new WebPubSubServiceClient(new Uri(endpoint), "some_hub", ne
 
 ### Connection
 
-Connections, represented by a connection id, represent an individual websocket connection to the Web PubSub service. Connection id is always unique.
+A connection, also known as a client or a client connection, represents an individual WebSocket connection connected to the Web PubSub service. When successfully connected, a unique connection ID is assigned to this connection by the Web PubSub service.
```

Comment:
If I understand this right, you are saying that a connection can also be known as either 'client' or 'client connection'. Is that right?
I am not sure of the absolute equivalence between a 'connection' and a 'client'. Can you interchange the two words in all contexts for this service?
Also, could you add quotes around 'client connection' to clarify that the 2-word phrase is the replacement.

Reply:
under the Web PubSub service context, "client" only stands for the "WebSocket client", so I try to use "serviceClient" when using this libarary to avoid confusion

Reply:
Add a diagram trying to illustrate the case. Noticed that CI has a hard requirement about the title to use "Authenticate the [Cc]lient", can we suppress that?